src/emcommon/diary/base_mode_colors.py

```diff
@@ -0,0 +1,43 @@
+modeColors = {
+    "pink": '#c32e85', # oklch(56% 0.2 350)     # e-car
+    "red": '#c21725', # oklch(52% 0.2 25)      # car
+    "orange": '#bf5900', # oklch(58% 0.16 50)     # air, hsr
+    "green": '#008148', # oklch(53% 0.14 155)    # bike, e-bike, moped
+    "blue": '#0074b7', # oklch(54% 0.14 245)    # walk
+    "periwinkle": '#6356bf', # oklch(52% 0.16 285)    # light rail, train, tram, subway
+    "magenta": '#9240a4', # oklch(52% 0.17 320)    # bus
+    "grey": '#555555', # oklch(45% 0 0)         # unprocessed / unknown
+    "taupe": '#7d585a', # oklch(50% 0.05 15)     # ferry, trolleybus, user-defined modes
+}
+
+BaseModes = {
```

Comment:
```suggestion
BASE_MODES = {
```
uppercase SNAKE_CASE to signify that this is a constant